twilight-lavalink/src/model/incoming.rs

```diff
@@ -0,0 +1,373 @@
+    /// retrieved via /v4/stats.
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub frame_stats: Option<StatsFrame>,
+    /// Memory information about the node's host.
+    pub memory: StatsMemory,
+    /// The current number of total players (active and not active) within
+    /// the node.
+    pub players: u64,
+    /// The current number of active players within the node.
+    pub playing_players: u64,
+    /// The uptime of the Lavalink server in seconds.
+    pub uptime: u64,
+}
+
+/// CPU information about a node and its host.
+#[derive(Clone, Debug, Deserialize, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct StatsCpu {
+    /// The number of CPU cores.
+    pub cores: usize,
+    /// The load of the Lavalink server.
+    pub lavalink_load: f64,
+    /// The load of the system as a whole.
+    pub system_load: f64,
+}
+
+/// CPU information about a node and its host.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct StatsFrame {
+    /// The load of the system as a whole.
+    pub deficit: i64,
+    /// The load of the Lavalink server.
+    pub nulled: i64,
+    /// The number of CPU cores.
+    pub sent: i64,
+}
+
+/// Memory information about a node and its host.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct StatsMemory {
+    /// The number of bytes allocated.
+    pub allocated: u64,
+    /// The number of bytes free.
+    pub free: u64,
+    /// The number of bytes reservable.
+    pub reservable: u64,
+    /// The number of bytes used.
+    pub used: u64,
+}
+
+/// Information about the track returned or playing on Lavalink.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct TrackInfo {
+    /// The track artwork url.
+    pub artwork_url: Option<String>,
+    /// The track author.
+    pub author: String,
+    /// The track [ISRC](https://en.wikipedia.org/wiki/International_Standard_Recording_Code).
+    pub isrc: Option<String>,
+    /// The track identifier.
+    pub identifier: String,
+    /// Whether the track is seekable.
+    pub is_seekable: bool,
+    /// Whether the track is a stream.
+    pub is_stream: bool,
+    /// The track length in milliseconds.
+    pub length: u64,
+    /// The track position in milliseconds.
+    pub position: u64,
+    /// The track source name.
+    pub source_name: String,
+    /// The track title.
+    pub title: String,
+    /// The track uri.
+    pub uri: Option<String>,
+}
+
+/// A track object for lavalink to consume and read.
+#[derive(Clone, Debug, Deserialize, Eq, PartialEq, Serialize)]
+#[non_exhaustive]
+#[serde(rename_all = "camelCase")]
+pub struct Track {
+    /// The base64 encoded track to play
+    pub encoded: String,
+    /// Info about the track
+    pub info: TrackInfo,
+}
```

Comment:
[no action needed] The docs also list `pluginInfo`, `userData` though I can understand if it is intentionally that you have not listed them here since they are just "`objects`".